researchgraph/writingnode/writeup.py

```diff
@@ -1,17 +1,15 @@
 import os
 import os.path as osp
-import json
-from typing import Any, Optional
+from typing import Any, Optional, TypedDict
 from aider.coders import Coder
 from aider.models import Model
 from aider.io import InputOutput
 from dataclasses import dataclass
 from researchgraph.graph.ai_scientist.ai_scientist_node.generate_ideas import search_for_papers
```

Comment:
Because it is difficult to handle as a single node, please avoid importing it from a completely different script.